snuba/subscriptions/scheduler.py

```diff
@@ -278,6 +283,43 @@ def add_tag(tags: Tags, builder_type: str) -> Tags:
         ]
 
 
+def filter_subscriptions(
+    subscriptions: List[Subscription],
+    entity_key: EntityKey,
+    metrics: MetricsBackend,
+    slice_id: Optional[int] = None,
+) -> List[Subscription]:
+    filtered_subscriptions: List[Subscription] = []
+
+    # only for storage sets that are currently sliced
+    entity = get_entity(entity_key)
+    storage = entity.get_writable_storage()
+    if storage is not None:
+        storage_set_key = storage.get_storage_set_key()
+
+        if storage_set_key.value in settings.SLICED_STORAGE_SETS:
+            for subscription in subscriptions:
+                # get the metadata and org_id from the Subscription
+                sub_data = subscription.data
+                sub_metadata = sub_data.metadata
+                org_id = sub_metadata["organization"]
+
+                if org_id is not None:
+                    # map the org_id to the slice ID
+                    logical_part = map_org_id_to_logical_partition(org_id)
+                    part_slice_id = map_logical_partition_to_slice(
+                        storage.get_storage_set_key(), logical_part
+                    )
+                    if part_slice_id == slice_id:
+                        filtered_subscriptions.append(subscription)
+                else:
+                    metrics.increment("queries_with_orgID=None")
```

Comment:
I don't know if datadog supports using `=` in tag names. But in general, you should avoid using it. You can rename this to something like `subscription_with_empty_org_id`

Reply:
agree with this